scripts/qserver_screen.sh

```diff
@@ -5,10 +5,11 @@
 
 #--------------------
 # change the program defaults here
-source "${HOME}/.bash_aliases"
+
+source "${HOME}/.bash_aliases" #not included on the package, must be deleted
```

Comment:
Agreed.  Probably provided some env vars assumed below.  If undefined, they evaluate to `""`, so we need to be aware of that possibility.